tests/behaviour/contracts/conversions/widthsignWideningItoUchange.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity ^0.7.6;
+
+//SPDX-License-Identifier: MIT
+
+contract WARP {
+    function widthsignWitou(int16 x) pure public returns (uint32) {
+    return (uint32(x));
+  }
+}
```

Comment:
Could you add newlines at the end of all the files that have this symbol at the end